### PR TITLE
Added solution to "Responsive Web design" challenge 

### PR DESCRIPTION
**Pull request by CubeSkyy**
- [X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] None of my changes are plagiarized from another source without proper attribution.
- [X] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [X] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md [hunk 1]
```diff
@@ -81,7 +81,41 @@ tests:
 ## Solution
 <section id='solution'>
 
-```js
-// solution required
+```html
+<style>
+h2 {color: blue;}
```
Comment (huyenltnguyen) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: The code works, but I think we should make it more readable:
```suggestion
  h2 {
    color: blue;
  }
```

Comment (CubeSkyy) [reply curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: Yeah. Thanks for the feedback!

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍 

Thanks @CubeSkyy

**Comment by RandellDawson** [comment]
@huyenltnguyen @CubeSkyy This PR needs to be reverted, because the solution actually fails at least the following test (and maybe others):
```yml
  - text: Crie um elemento de <code>style</code> .
    testString: 'assert($("style") && $("style").length > 1, "Create a <code>style</code> element.");'
```

**Pull request merged**